friendslist/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Friends-list</title>
+
+    <link href="css/index.css" rel="stylesheet">
```

Comment:
use `./` to start with the relative path

Reply:
changed it to relative path